dryville_story.md

```diff
@@ -15,3 +15,22 @@ So let's say you made the big move. You and some friends have found your (desola
+
+The problem is, the houses are higher than the creek, and water does not flow uphill. [Water does flow downhill](https://www.usgs.gov/special-topic/water-science-school/science/watersheds-and-drainage-basins), so you build a big water storage tank on a hill (hopefully at the highest point in town) and establish a water brigade to fill it full of water. You can run pipes from the storage tank down to everyone's home. The pipes go right to your faucet and you use gravity (it's free!) to get the water delivered. Works great -- just like it did in the big city you came from.
+
+But you still have the problem of keeping the storage tank full. Lugging water up to it is no better than lugging it up to your house; so you need to find a way to pump the water from the [creek](https://www.usgs.gov/special-topic/water-science-school/science/rivers-streams-and-creeks) uphill into the storage tank. Since Dryville doesn't have any industry to produce products to sell to the outside world to bring in money to buy things with yet, you need to build your own pump. And you can't use electricity since you haven't built a power plant yet (but you will). Can you think of a source of energy that can run your pump? A windmill could do the job, but in the desert there isn't always wind. Your creek! The water in your creek is flowing nicely -- so you build a paddle wheel in the river. The paddle spins in the creek's current and turns a rod that runs your homemade pump. Voila! You've got water being pumped uphill into your storage tank where gravity lets it flow to your homes.
+
+##Dryville's First Water Works
+Now everyone is enjoying running water in their homes. But it took a lot of work to build your water-supply plant ("Horace's Water Works"). Being mayor, you decide that the town of Dryville will be the owner of the new Dryville Water Plant and you'll charge everyone to get water delivered to their houses. Since the Water Plant is owned by Dryville, it has to respond to the needs of its citizens -- such as Mr. Milford, who overcooked his possum stew and started a fire in his kitchen. "Where was the Dryville Fire Department when I needed them!" he complains to the mayor. Very well, you add some fire hydrants to the water-supply pipes and now you are the Fire Chief as well as mayor.
+
+Soon you start getting money from the citizens buying water from you - and that gives you an idea. You're going to write an acquaintance back home and tell him he needs to build another town down the road from Dryville. Then when your friend gets his town started, Dryville will offer to sell them water that you get from your public-supply system! You can build an [aqueduct system](https://www.usgs.gov/special-topic/water-science-school/science/aqueducts-move-water-past-and-today) to move water from Dryville to them. Of course, there will be a mark up to cover the cost of delivering the water — nothing wrong with making a little profit.
+
+##Be Gone, Dirty Water
```

Comment:
I think you want to delete this section. You should only have two in this first PR. This third section will come later in #10